apps/visualizer/plugins/createGitignore.ts

```diff
@@ -0,0 +1,24 @@
+import { writeFileSync } from 'fs';
+import type { PluginOption } from 'vite';
+import { normalizePath } from 'vite';
+
+/**
+ * The `dist` directory is deleted every time the bundle is rebuilt. The `.gitignore` file
```

Comment:
The dist folder doesn't need to be in git, just in the npm package when published.

Reply:
Alright